CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Changelog
 
+## Version 0.2.2
+
+[diff since 0.2.0](https://github.com/QEDjl-project/QEDbase.jl/compare/release-0.2.0...release-0.2.2)
+
+This release adds some convenience overloads to existing functions, some code maintenance and small fixes.
+
+### Breaking Changes
+
+This release removes the compute setup interface completely since it was deprecated already.
+See https://github.com/QEDjl-project/QEDbase.jl/pull/91 for details.
+
+### New features
```

Comment:
Would you mind adding references to the respective PR for the changes? The milestone `Release-v0.2.2` might come in handy.

Reply:
nice! Thanks.